package.json

```diff
@@ -24,6 +24,7 @@
   },
   "main": "dist/vast-client-node.min.js",
   "module": "src/index.js",
+  "types": "src/index.d.ts",
```

Comment:
> Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so.

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package